charts/atlas-cluster/templates/atlas-cluster.yaml

```diff
@@ -3,7 +3,7 @@
 apiVersion: atlas.mongodb.com/v1
 kind: AtlasCluster
 metadata:
-  name: {{ .name }}
```

Comment:
This change will create multiple clusters (I think `.Values.clusters` is an array), with the same name, because `$.Release.Name` is the same for all of them.
If we need this, then we need to make `clusters` 1 value (not a list of them)

Reply:
Add `.name` here to align with `.username` appending to the Helm release name.